src/main/resources/locales/en/accessibility-statement.json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Accessibility statement for the Court and Tribunal Hearing Information Service",
+  "section1": {
+    "text1": "The Court and Tribunal Hearing Information Service provides a publishing platform to enable information provided by HMCTS to be shared, displayed and updated as appropriate. It will enable lists, outcomes, judgments and information to be displayed to court users, consistent with jurisdictional procedures and business rules. The service supports HMCTS’ commitment to open justice and the provision of transparent and consistent court/tribunal information across all jurisdictions.",
+    "text2": "This accessibility statement applies to content published on ",
+    "text3": " It is based on an accessibility audit for carried out by the Digital Accessibility Centre (DAC) user/ technical team on 1 March 2022.",
```

Comment:
I think the spacing on user/ technical here needs updating too (same fix as the one further down)

Reply:
Done :)